### ISSUE TITLE
groups/talk/landscape: add safari-specific favicon files

### ISSUE THREAD

**Issue by urcades** [tloncorp/landscape-apps#1072]
Each of our apps do not implement an [apple touch icon](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html), which is necessary to display favicon-like image in safari (across both macOS and iOS).

Right now our apps look like this (note the tabs with only letters demarcating what they are):

<img width="905" alt="Screen Shot 2022-11-03 at 12 51 38 PM" src="https://user-images.githubusercontent.com/1195363/199785011-e1fd6eb8-1283-4f30-a6ec-fd0486fce595.png">

Basically, let's be sure to include these in coming updates so our apps are able to be distinguished easier on the mentioned platforms.

**Comment by jamesacklin** [comment]
Would like to add that this would address my idiosyncratic irritation: 

![image](https://user-images.githubusercontent.com/748181/199844489-fbdcc67e-c04f-4852-be3d-62063d78cbae.jpeg)

I am a progressive web app guy through and through; this would bring me (personally) one step closer to nirvana.
